### PR TITLE
{lib}[foss/2018b] python-igraph v0.7.1.post6, igraph v0.7.1, PyCairo v1.18.0 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/i/igraph/igraph-0.7.1-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+sources = ['igraph-%(version)s.tar.gz']
+checksums = ['d978030e27369bf698f3816ab70aa9141e9baf81c56cc4f55efbe5489b46b0df']
+
+builddependencies = [('Autotools', '20180311')]
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('libxml2', '2.9.8'),
+]
+
+preconfigopts = "aclocal && autoconf && automake && libtoolize && "
```
Comment (akesandgren) [inline easybuild/easyconfigs/i/igraph/igraph-0.7.1-foss-2018b.eb]: Wouldn't autoreconf do here?

Comment (boegel) [reply easybuild/easyconfigs/i/igraph/igraph-0.7.1-foss-2018b.eb]: Based on https://linux.die.net/man/1/autoreconf, it probably should...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3169.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/d3be82d032ab7cab900719da81c9fdcf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2490.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/01ebc4ee9a834259f66c32d06a755160 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2032.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f54119dff8f93311e6c08959768980f8 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**